packages/office-ui-fabric-react/src/components/Pivot/Pivot.tsx

```diff
@@ -118,7 +118,7 @@ export class Pivot extends React.Component<IPivotProps, IPivotState> {
         role='tab'
         aria-controls={ id + '-panel' }
         aria-selected={ this.state.selectedKey === itemKey }>
-        { link.linkText }
+        <span className='ms-Pivot-text'>{ link.linkText }</span>
         { countText }
```

Comment:
@dzearing: This is resolved, right? I'll let you merge the PR in case I'm missing a request here.